test/functional/test_runner.py

```diff
@@ -621,6 +623,19 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, attempts=1, enab
             done_str = f"{len(test_results)}/{test_count} - {BOLD[1]}{test_result.name}{BOLD[0]}"
             if test_result.status == "Passed":
                 logging.debug("%s passed, Duration: %s s" % (done_str, test_result.time))
+                # Remove blocks folder from test datadir to free up CI disk space
```

Comment:
why that doesn't happen with bitcoin? they have no cleanup mechanism on CI

Btw, I just noticed that all these blocks are not sparsed files, but regular files fullfilled zeroes.
So, they are compressed well when CI is done; but during a run they are eating disk space.

```
/** The pre-allocation chunk size for blk?????.dat files (since 0.8) */                              
static const unsigned int BLOCKFILE_CHUNK_SIZE = 0x1000000; // 16 MiB                                
```

3 nodes for each test - 48MiB used; 300 tests -> 10+ Gb of zeroes (probably even more)


I guess; this fix is going to work anyway; but is it a bug or expected behaviour that blk files are not sparsed?

Reply:
```
$ test/functional/rpc_help.py
<succeed>
$ find /tmp/dash_func_test_5coohz_8 -ls | grep blk 
   436673  16384 -rw-------   1 knst     knst     16777216 Nov 17 02:08 /tmp/dash_func_test_5coohz_8/node0/regtest/blocks/blk00000.dat
$ du -h /tmp/dash_func_test_5coohz_8/node0/regtest/blocks/blk00000.dat
16M     /tmp/dash_func_test_5coohz_8/node0/regtest/blocks/blk00000.dat
```

Reply:
```
diff --git a/src/node/blockstorage.h b/src/node/blockstorage.h
index b627c26162..46e253b8aa 100644
--- a/src/node/blockstorage.h
+++ b/src/node/blockstorage.h
@@ -39,7 +39,7 @@ static constexpr bool DEFAULT_STOPAFTERBLOCKIMPORT{false};
 static constexpr bool DEFAULT_TIMESTAMPINDEX{false};
 
 /** The pre-allocation chunk size for blk?????.dat files (since 0.8) */
-static const unsigned int BLOCKFILE_CHUNK_SIZE = 0x1000000; // 16 MiB
+static const unsigned int BLOCKFILE_CHUNK_SIZE = 0x400000; // 4 MiB
```

What are possible downsides of this change?
I tried to run functional tests locally and it seems as IO become a smaller limiting factor; because funcitonal tests running as 30 parallel jobs (-j30) speeded up from 195s to just 180s

Reply:
`pruning` probably will break many functional tests, including governance's related; blockfilter, etc

But we could one more param to use here to use small blocks, something like :
```
    return FlatFileSeq(gArgs.GetBlocksDirPath(), "blk", 
        gArgs.GetBoolArg("-tinyblk", false) ? 0x10000 (64kB)
        : (gArgs.GetBoolArg("-fastprune", false) ? 0x4000 /* 16kb */ : BLOCKFILE_CHUNK_SIZE));
```